src/components/Dnd/Dnd.js

```diff
@@ -0,0 +1,145 @@
+import React, { useEffect, useState } from "react";
+import styled from "styled-components";
+
+import { dragOver, dragStart, dragEnd, drop } from "../../utils/dnd";
+import { buildTree } from "../../utils/tree";
+import { fetchCreatedFolder, moveFolder } from "../../redux/slices/folderSlices";
+import { useDispatch, useSelector } from "react-redux";
+
+function Dnd() {
+  const dispatch = useDispatch();
+  const folderList = useSelector(state => state.folder.folderList);
+  const [ grabFolder, setGrabFolder ] = useState(null);
+
+  useEffect(() => {
+    dispatch(fetchCreatedFolder());
+  }, []);
```

Comment:
```suggestion
  }, [dispatch]);
```
저의 경우 dispatch에 대한 의존성 경고가 있었어서 추가해주시면 좋을 것 같습니다!

Reply:
말씀대로 의존성 경고가 떠서 수정했습니다.